### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by catalin-ilea**
https://xp-up.ft.com/__health?categories=lists-publish
https://xp-up.ft.com/__health?categories=lists-read

### CODE REVIEW

**Comment by catalin-ilea** [comment]
Looks like we have a 🐛 in the agg hc around categories. Looking into it.

**Comment by catalin-ilea** [comment]
Not a bug, me being silly. Updating PR now.

File: aggregate-healthcheck-sidekick@.service [hunk 1]
```diff
@@ -10,6 +10,10 @@ ExecStart=/bin/sh -c "\
   etcdctl mkdir /ft/services/$SERVICE/servers; \
   etcdctl set /ft/healthcheck-categories/read/period_seconds 4; \
   etcdctl set /ft/healthcheck-categories/read/is_resilient true; \
+  etcdctl set /ft/healthcheck-categories/lists-read/period_seconds 60; \
+  etcdctl set /ft/healthcheck-categories/lists-read/is_resilient false; \
```
Comment (tosan88) [inline aggregate-healthcheck-sidekick@.service]: isn't read resilient?

**Comment by martingartonft** [comment]
lgtm

**Comment by tosan88** [comment]
:+1:

File: aggregate-healthcheck-sidekick@.service (hunk 1 above)
Comment (catalin-ilea) [inline aggregate-healthcheck-sidekick@.service]: This won't be configured in DYN or anything like that. It's just a hc for all the services that might impact lists-read pipeline.

**Pull request merged**